app/Makefile

```diff
@@ -7,3 +7,11 @@ release-build:
 		--platform=linux/amd64 \
 		$(OPTS) \
 		.
+
+init:
+	@echo "Example command for ci-vulnerability-scans.yml pipeline to build Docker image"
+	@echo "If you are using this file, update this command to build your app for testing"
+
+testing:
+	@echo "Example command for ci-vulnerability-scans.yml pipeline to run tests"
+	@echo "This is where any testing for your CI would go"
```

Comment:
This is also covered by the application (see https://github.com/navapbc/template-infra/blob/main/template-only-docs/application-requirements.md).

and as an example see https://github.com/navapbc/template-application-nextjs/blob/main/app/Makefile and https://github.com/navapbc/template-application-flask/blob/main/app/Makefile which both define `make release-build` in `app/Makefile`, so we don't need it here.